<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettings</class>
 <widget class="QWidget" name="CameraSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Camera type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_camera_type">
       <property name="toolTip">
        <string extracomment="Select camera type">Select camera type</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Displayed name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name">
       <property name="toolTip">
        <string>Under this name camera will be displayed in viewer</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_enabled">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="fr_tango">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QCheckBox" name="chk_manual_tango_device">
        <property name="text">
         <string>Specify manually:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="cmd_rescan_database">
        <property name="text">
         <string>Rescan database</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="cmb_tango_device"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_tango_host">
        <property name="toolTip">
         <string>Type tango database host, e.g. hasep23oh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>TangoDB host:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Select camera:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="le_tango_server">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Camera tango device, e.g. pXX/lmscreen/LM_XX</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="cmb_roi_device"/>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="le_roi_server">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Camera tango device, e.g. pXX/lmscreen/LM_XX</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="chk_manual_roi_device">
        <property name="text">
         <string>Specify manually:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chk_roi_server">
        <property name="text">
         <string>ROI server</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Screen motor type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_motor_type">
       <property name="whatsThis">
        <string>If screen has a tango motor</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="fr_motor">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="fr_fsbt">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>FSBT server host</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_fsbt_host"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>FSBT server post</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_fsbt_port"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>FSBT motor name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_fsbt_motor_name"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_acromag">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Valve tango server</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_acromag_server"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Valve channel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_acromag_valve"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Picture settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chk_flip_v">
        <property name="text">
         <string>Flip vertical</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chk_flip_h">
        <property name="text">
         <string>Flip horizontal</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Rotate in 90 deg step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sp_rotate">
          <property name="maximum">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chk_color">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chk_high_depth">
        <property name="text">
         <string>High depth</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
